server/index.js

```diff
@@ -21,9 +20,6 @@ app.use(morgan('dev'));
 app.use(bodyParser.urlencoded({ extended: false }));
 app.use(bodyParser.json());
 
-app.use(methodOverride());
-app.use(express.static(`${__dirname}/public`));
-
 app.use(
   session({
     secret: 'authorshaven',
```

Comment:
Chima! Please return the whole session script as it was. It was already set to false

Reply:
That is not a session script. @Oluwafayokemi. And the folder you are referring to don't exist.